device-registry/src/main/scala/org/genivi/sota/device_registry/GroupsResource.scala

```diff
@@ -100,7 +99,7 @@ class GroupsResource(namespaceExtractor: Directive1[AuthedNamespaceScope], devic
       (scope.get & pathEnd) {
         listGroups(ns)
       } ~
-      (scope.get & extractGroupId & path("devices") ) { groupId =>
+      (scope.get & extractGroupId & path("devices") & pathEnd ) { groupId =>
```

Comment:
It is used in other endpoints that use parameters, but it seems like it isn't needed.